mappings/net/minecraft/entity/player/HungerConstants.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_6075 net/minecraft/entity/player/HungerConstants
+	COMMENT Some constants on hunger values.
+	COMMENT
+	COMMENT @see HungerManager
+	FIELD field_30704 FULL_FOOD_LEVEL I
+		COMMENT The maximum food level ({@value}) allowed in a hunger manager.
+	FIELD field_30706 INITIAL_SATURATION_LEVEL F
+		COMMENT The initial saturation level ({@value}) for a newly created hunger manager.
+	FIELD field_30708 EXHAUSION_UNIT F
+		COMMENT A value {@value} that when the exhausion reaches, the exhausion minuses itself
+		COMMENT by to reduce the saturation or food level.
```

Comment:
```suggestion
	FIELD field_30708 MAX_EXHAUSTION
		COMMENT The value {@value} at which exhaustion resets
		COMMENT causing saturation or hunger to decrease.
```

Reply:
It doesn't reset. I chose your wording initially but decided to use the current one because the exhausion may be 6 and it will decrease by 4 and become 2 than reset to 0.
```suggestion
	FIELD field_30708 EXHAUSTION_UNIT F
		COMMENT A value {@value} that when the exhaustion reaches, the exhaustion minuses itself
		COMMENT by, causing reduction in saturation or food level.
```